app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  after_create :update_comments_counter
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  def update_comments_counter
+    Post.find(post.id).increment!(:comments_counter)
+  end
```

Comment:
- [ ] Great job making use of [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html). But when ever a `callback` is used, the method that uses it is made `private`. Kindly check the example below using the callback `after_save` and fix your code accordingly. 

```ruby
after_save :update_comments_counter

private
def update_comments_counter
  post.increment!(:comments_counter)
end
```